### PR TITLE
Vulcan: Add featured blockquote styling

### PR DESCRIPTION
**Pull request by aburke07**
This matches the wiki view styling and applies to all featured blockquotes in Vulcan.

We still need to allow block quotes in the wysiwyg editor, but using plaintext still lets us add a featured block quote to wikis, so this won't break anything.

## QA
1. Open a vulcan wiki and add a featured block quote to the wiki text.
- Ex: https://me.cominor.com/Wiki/Edit/Dryer_Stops_Mid_Cycle
- Add this text: `[quote|format=featured]This is a featured blockquote! -Alex[/quote]`
2. Viewing the Vulcan wiki should render the block quote with this styling:
- Ex: http://localhost:3000/Vulcan/Dryer_Stops_Mid_Cycle
image

Part 1 of 2 for https://github.com/iFixit/ifixit/issues/48526

### CODE REVIEW

**Review by ianrohde — COMMENTED** [review]
We've got `margin-top: var(--chakra-space-4);` being applied to all `<p>` elements. Can we do something like: `blockquote > p { marginBlock: 0}` to patch them in one pass?

File: frontend/templates/troubleshooting/prerendered.tsx [hunk 1]
```diff
@@ -75,6 +75,12 @@ const renderStyles: SystemStyleObject = {
       borderLeftWidth: '5px',
       borderLeftStyle: 'solid',
       padding: '2px 8px 2px 12px',
+
+      '&.featured': {
+         borderColor: '#fe6f15',
+         borderTopWidth: '1px',
+         borderBottomWidth: '1px',
```
Comment (ianrohde) [inline frontend/templates/troubleshooting/prerendered.tsx:82]:
```suggestion
         borderBottomWidth: '1px',
         marginBlock: '12px',
```
This vertically centers and overwrites the margin-top: var(--chakra-space-4); being applied to all <p> elements.

File: frontend/templates/troubleshooting/prerendered.tsx [hunk 2]
```diff
@@ -75,6 +75,12 @@ const renderStyles: SystemStyleObject = {
...
+      },
```
Comment (ianrohde) [inline frontend/templates/troubleshooting/prerendered.tsx:83]:
```suggestion
      },
      
      '& > p': {
         marginBlock: '8px',
      },
```

**Review by ianrohde — APPROVED** [review]
CR ⚪

**Comment by deltuh-vee** [comment]
QA 🎬 
Vulcan pages can now render featured block quotes

**Comment by aburke07** [comment]
QA ⏫ Carryover: just prettier changes

**Review by ianrohde — APPROVED** [review]
CR ⚪

**Comment by deltuh-vee** [comment]
Carry Over QA 🎬

**Pull request merged**